packages/client/src/routes/Room/Room.tsx

```diff
@@ -15,9 +14,30 @@ const Header = styled.div`
   width: 100%;
 `
 
+interface ChatMember {
+  name: String
+  id: String
+}
+
 const Room: React.FC = () => {
   const [message, setMessage] = React.useState("")
   const { roomId } = useParams()
+  const navigate = useNavigate()
+  const [chatMember, setChatMember] = React.useState<ChatMember>({
+    name: "",
+    id: "",
+  })
+
+  React.useEffect(() => {
+    if (!localStorage.getItem("member_id")) {
```

Comment:
I would set the localstorage calls as variables for increased readability and one of them is ised multiple times

Reply:
Agree. Maybe make a utils file? localStorage.setItem can actually throw an error too, so wrap it in a try/catch. I think there is code that does this in gavagai, for reference :)

Reply:
I will look in to that!